Find a functionally equivalent code:android.os.Bundle.putShort:COMMENT
Method Modifier: public      
Comment:/**
 * Inserts a short value into the mapping of this Bundle, replacing
 * any existing value for the given key.
 *
 * @param key a String, or null
 * @param value a short
 */

@@ -1,4 +1,3 @@
 {
-    unparcel();
-    mMap.put(key, value);
+    super.putShort(key, value);
 }

